recipes/bw2speedups/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "bw2speedups" %}
+{% set version = "2.2" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "04cf55d9fdb74876fb690978966ca5901f2908a8d7607feb99b9e5fe361b8f72" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
The newer convention is to use `pip`:

```
script: python -m pip install --no-deps --ignore-installed .
```

you would then need to add `pip` as a `build` dependency.